AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginFunctionalTests/GraphQLConnectionScenario3Tests+Subscribe.swift

```diff
@@ -48,13 +48,13 @@ extension GraphQLConnectionScenario3Tests {
         }
         
         XCTAssertNotNil(task)
-        try await AsyncExpectation.waitForExpectations([connectedInvoked], timeout: TestCommonConstants.networkTimeout)
+        await waitForExpectations([connectedInvoked], timeout: TestCommonConstants.networkTimeout)
         let post = Post3(id: uuid, title: title)
         _ = try await Amplify.API.mutate(request: .create(post))
         let post2 = Post3(id: uuid2, title: title)
         _ = try await Amplify.API.mutate(request: .create(post2))
 
-        try await AsyncExpectation.waitForExpectations([progressInvoked], timeout: TestCommonConstants.networkTimeout)
+        await waitForExpectations([progressInvoked], timeout: TestCommonConstants.networkTimeout)
         await task.cancel()
```

Comment:
i see a warning `No 'async' operations occur within 'await' expression` at this line in this file. is this something that we can remove?